pkg/bisect/bisect.go

```diff
@@ -160,7 +160,7 @@ func runImpl(cfg *Config, repo vcs.Repo, bisecter vcs.Bisecter, inst instance.En
 	}
 	com := res.Commits[0]
 	env.log("first %v commit: %v %v", what, com.Hash, com.Title)
-	env.log("cc: %q", com.CC)
+	env.log("cc: %q", com.Recipients.GetEmails(true))
```

Comment:
In general, please keep the code up-to-date.  In this case, shouldn't "cc:" be replaced with "recipients:"?